src/Symfony/Component/Messenger/Tests/Transport/SqsExt/SqsExtIntegrationTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Tests\Transport\SqsExt;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Tests\Fixtures\DummyMessage;
+use Symfony\Component\Messenger\Transport\SqsExt\Connection;
+
+class SqsExtIntegrationTest extends TestCase
+{
+    private $connection;
+
+    protected function setUp(): void
+    {
+        if (!getenv('MESSENGER_SQS_DSN')) {
+            $this->markTestSkipped('The "MESSENGER_SQS_DSN" environment variable is required.');
+        }
+
+        $this->connection = Connection::fromDsn(getenv('MESSENGER_SQS_DSN'), []);
+        $this->connection->setup();
+        $this->clearSqs();
+    }
+
+    public function testConnectionSendAndGet()
+    {
+        $this->connection->send('{"message": "Hi"}', ['type' => DummyMessage::class]);
+        $wait = 0;
+        while ((null === $encoded = $this->connection->get()) && $wait++ < 200) {
+            usleep(5000);
+        }
+
+        $this->assertEquals('{"message": "Hi"}', $encoded['body']);
+        $this->assertEquals(['type' => DummyMessage::class], $encoded['headers']);
+    }
+
+    private function clearSqs()
+    {
+        $this->connection->requeuePrefetchedMessages();
```

Comment:
Don't you actually want to not requeue them and simply delete them?

Reply:
This variable is a buffer containing "unproceed" messages. 
No we don't want to delete theme. We want to mark theme as "ready to be fethed" in order to let another consumer pick theme

Reply:
I missed that your comment was in a Test class.
Indeed, this is useless as we want to remove all of those messages